examples/_includes/directory_list.xml.njk

```diff
@@ -1,13 +1,13 @@
 {# Standard template for showing a list of links to other screens. List is generated from a collection of screens with the given "list_tag" #}
-{% extends 'base.xml.njk' %}
 
-{% block styles %}
-
-{% endblock %}
-
-{% block content %}
-  <behavior trigger="refresh" action="reload" href="#" />
-  <list>
+<list
+  xmlns="https://hyperview.org/hyperview"
+  trigger="refresh"
+  action="replace"
```

Comment:
I think the original screens were written before we had the `reload` action. With the addition of this action, I think we don't need to override `list_href` for each directory, or have a separate list template, right? That would simplify this work quote a bit.

Reply:
The `reload` action on lists (paired with `href="#"`) does not work expected - the loading state remains stuck:

https://user-images.githubusercontent.com/309515/179427537-bfbf6adc-1e42-4926-9702-67e4ed254689.mp4

Reply:
I see. You can also try to use the eleventy metadata, something like:
`href="{{ permalink }}"` instead of `href="{% block list_href %}{% endblock %}"`.

Reply:
I think the issue will be that this will generate a URL to the whole screen, not the URL of the list partial

Reply:
I see I can define another variable but it looks like it would needs to be defined in both `index.xml.njk` and `list.xml.njk`. This is how the code change looks like:
```diff
diff --git a/examples/_includes/directory_list.xml.njk b/examples/_includes/directory_list.xml.njk
index eb4a793..6e42e2b 100644
--- a/examples/_includes/directory_list.xml.njk
+++ b/examples/_includes/directory_list.xml.njk
@@ -6,7 +6,7 @@
   action="replace"
   target="directory-list"
   id="directory-list"
-  href="{% block list_href %}{% endblock %}"
+  href="{{ list_href }}"
 >
   {% for screen in collections[list_tag] %}
     <item
diff --git a/examples/index.xml.njk b/examples/index.xml.njk
index e4a1f38..8e5cec0 100644
--- a/examples/index.xml.njk
+++ b/examples/index.xml.njk
@@ -2,7 +2,8 @@
 title: "Hyperview"
 permalink: "index.xml"
 list_tag: "root"
-hide_back_button: true,
+hide_back_button: true
+list_href: "list.xml"
 ---
 {% extends 'base.xml.njk' %}

diff --git a/examples/list.xml.njk b/examples/list.xml.njk
index c36f88e..7b98f5b 100644
--- a/examples/list.xml.njk
+++ b/examples/list.xml.njk
@@ -1,7 +1,6 @@
 ---
 permalink: "list.xml"
 list_tag: "root"
+list_href: "list.xml"
 ---
 {% extends "directory_list.xml.njk" %}
-
-{% block list_href %}list.xml{% endblock %}
```

Reply:
Oh yeah that could work, I was under the impression if statements did not exist in njk templates. Will try that.